test/feeds.test.js

```diff
@@ -133,6 +134,88 @@ describe('test POST /feeds endpoint', () => {
   });
 });
 
+describe('test PUT /feeds/cache endpoint', () => {
+  it('should respond with a 403 status trying to clear cache without logging in', async () => {
+    const res = await request(app).put('/feeds/cache').send().set('Accept', 'application/json');
```

Comment:
`delete` vs. `put` (same for all tests below)

Reply:
Okay, I'm confused about this part since I wrote it as a PUT on line 78 of `feeds.js`

Reply:
We could probably argue it both ways.  I think `DELETE` is more accurate, since we're removing something.  With `PUT`, you'd expect some data to accompany the request, as in "here are the new value(s)".

Reply:
Okay, I will update the route to use `DELETE` then. Thanks!